backend/substrapp/serializers/ledger/objective/serializer.py

```diff
@@ -16,6 +17,7 @@ class LedgerObjectiveSerializer(serializers.Serializer):
     test_data_manager_key = serializers.CharField(max_length=256, allow_blank=True, required=False, allow_null=True)
     permissions = PermissionsSerializer()
     metrics_name = serializers.CharField(min_length=1, max_length=100)
+    metadata = DictField(child=CharField(), required=False, allow_null=True, allow_empty=True)
```

Comment:
This serializer uses allow_empty=True unlike all other serializers. Should it be removed?

Reply:
Yes indeed, thanks!